.readthedocs.yml

```diff
@@ -7,7 +7,9 @@ version: 2
 
 # Specify docker image for building the doc
 build:
-  image: latest
+  os: ubuntu-22.04
```

Comment:
Why not latest here ?

Reply:
As per Read the docs documentation, this is a way to specify the build image in version 2 of configure file. 
Documentation link: https://docs.readthedocs.io/en/stable/config-file/v2.html#build-os